helix-term/src/keymap.rs

```diff
@@ -486,7 +486,6 @@ impl Default for Keymaps {
                 "a" => code_action,
                 "'" => last_picker,
                 "d" => { "Debug"
-                    "s" => dap_start,
                     "b" => dap_toggle_breakpoint,
```

Comment:
I removed the start shortcut for now. Two options are considered:
- Launch from command history
- Keep a `launch history` last element of which would be ran by a keybinding.

Reply:
Restored!